pkg/controller/jobframework/integrationmanager.go

```diff
@@ -288,6 +288,11 @@ func GetIntegrationsList() []string {
 	return manager.getList()
 }
 
+// GetEnabledIntegrationsList returns the list of currently enabled frameworks.
+func GetEnabledIntegrationsList() []string {
+	return manager.getEnabledIntegrations().UnsortedList()
+}
```

Comment:
```suggestion
// GetEnabledIntegrationsListForTest returns the list of currently enabled frameworks.
func GetEnabledIntegrationsList(tb *testing.T) []string {
	tb.Helper()
	return manager.getEnabledIntegrations().UnsortedList()
}
```

Isn't this only for testing? In that case, can you add "Test" suffix and receive testing object similar to "EnableIntegrationsForTest".

Additionally, could we avoid implementing such only testing functions? Is there any alternative solution?

Reply:
Just now it's using only for tests. But it can be useful in another places like `GetIntegrationsList`.

Reply:
Done